### ISSUE TITLE
stateChanged is never called

### ISSUE THREAD

**Issue by 1TheMuffinMan** [JustMaier/angular-signalr-hub#88]
I've followed the example in the readme regarding state changed and none of the event handlers are ever called. Any idea?

```
 stateChanged: function(state){
           console.log('signalR state changed');
            switch (state.newState) {
                case $.signalR.connectionState.connecting:
                    console.log('signalR connecting');
                    break;
                case $.signalR.connectionState.connected:
                    console.log('signalR connected');
                    break;
                case $.signalR.connectionState.reconnecting:
                    console.log('signalR reconnecting');
                    break;
                case $.signalR.connectionState.disconnected:
                    console.log('signalR disconnected');
                    break;
            }
        }
```

**Comment by 1TheMuffinMan** [comment]
The NPM package is not up to date which is where I got the source code from. Using the hub js file from this repository works.

**Comment by JustMaier** [comment]
Hey @1TheMuffinMan thanks for bringing this to my attention, looks like I had forgotten to bump the version after the last PR. Should be fixed now on v1.6.3
